### PR TITLE
wez.atomicparsley version 20221229.172126.d813aa6

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109505&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]
VC Redist is missing:
image

cc @stephengillie

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

waiting until dependency support is out of experimental features and is enabled in pipelines

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

File: manifests/w/wez/atomicparsley/20221229.172126.d813aa6/wez.atomicparsley.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: AtomicParsley.exe
+    PortableCommandAlias: AtomicParsley
+  InstallerSha256: 3F81DEFC8D52FFAE8520362725BB96ED31BE4FAEFE8BC25507E5B7CFE8CBF3C9
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (stephengillie) [inline manifests/w/wez/atomicparsley/20221229.172126.d813aa6/wez.atomicparsley.installer.yaml:26]:
```suggestion
```

File: manifests/w/wez/atomicparsley/20221229.172126.d813aa6/wez.atomicparsley.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    PortableCommandAlias: AtomicParsley
+  InstallerSha256: 39BEC5D5C0B33044D8ADF9F4C41F382DA090C61A597EE3543ED62A03B6CB56C9
+- InstallerUrl: https://github.com/wez/atomicparsley/releases/download/20221229.172126.d813aa6/AtomicParsleyWindowsX86.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: AtomicParsley.exe
+    PortableCommandAlias: AtomicParsley
+  InstallerSha256: 3F81DEFC8D52FFAE8520362725BB96ED31BE4FAEFE8BC25507E5B7CFE8CBF3C9
```
Comment (stephengillie) [inline manifests/w/wez/atomicparsley/20221229.172126.d813aa6/wez.atomicparsley.installer.yaml:22]:
```suggestion
  InstallerSha256: 3F81DEFC8D52FFAE8520362725BB96ED31BE4FAEFE8BC25507E5B7CFE8CBF3C9
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by stephengillie — APPROVED** [review]

**Pull request merged**